ramalama.py

```diff
@@ -32,22 +32,26 @@ def main(args):
         if host != port:
             port = host.rsplit(':', 1)[1]
 
+        syspath = '/usr/share/ramalama/python'
```

Comment:
Clever :)

I hope this works and we can exec this before we search for:

```
import errno
import os
from pathlib import Path
import subprocess
import ramalama
import sys
```

Reply:
We could always write a wrapper python script that did the equivalent of:

```
#!/bin/sh

export PYTHONPATH="/usr/share/ramalama/python:${PYTHONPATH}"
exec ramalama
```

that has basically no imports